### PR TITLE
[ADD] product_stock_on_hand: new module to take into account only the stock in hand

### PR DESCRIPTION
**Pull request by agaldona**

### CODE REVIEW

**Comment by agaldona** [comment]
@avanzosc/developers review when possible please.

File: product_stock_on_hand/models/stock_location.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+class StockLocation(models.Model):
+
+    _inherit = 'stock.location'
+
+    stock_on_hand = fields.Boolean(string='Stock On Hand')
+
+    @api.onchange('usage')
+    def onchange_usage(self):
+        self.ensure_one()
+        if self.usage == 'internal':
```
Comment (Daniel-CA) [inline product_stock_on_hand/models/stock_location.py]: self.stock_on_hand = self.usage == 'internal' or False

**Review by Daniel-CA — DISMISSED** [review]

**Review by Daniel-CA — APPROVED** [review]

**Review by alfredoavanzosc — APPROVED** [review]
👍

**Review by oihane — APPROVED** [review]
LGTM

**Pull request merged**